XenModel/Messages.resx

```diff
@@ -11124,6 +11124,9 @@ Do you want to connect to the pool master '{1}'?</value>
 
 You may need to reboot your server(s) to enable SR-IOV network.</value>
   </data>
+  <data name="SRIOV_NETWORK_SHOULD_BE_CREATED" xml:space="preserve">
+    <value>Yes (SR-IOV network should be created on this NIC to enable SR-IOV)</value>

```

Comment:
Let's make this a bit more user oriented: `Yes (To enable SR-IOV create an SR-IOV network on this NIC)`